README.md

```diff
@@ -100,40 +100,40 @@ export default class StepPlanet extends Steps {
 }
 ```
 
-The manager exposed his options to each steps, they can access it with `this.options`.
-The inheritance of the `Steps` class exposes following the class fields:
+The manager exposes its options on every step. Options can be accessed with `this.options`.
+The inheritance of the `Steps` class exposes the following class fields:
 
 #### `id`
 
 `String`
 
-The route identifier, an alias to used inside the app instead of the route.
+The route identifier is an alias to be used inside the app instead of the route.
 
-_The parameter is a public instance fields._
+_The parameter is a public instance field._
 
 #### `route`
 
 `String`
 
-The route for the step navigation. `StepManager` use hash for steps navigation (`#people`).
+The route for the step navigation. `StepManager` uses hashes for steps navigation (`#people`).
 
-_The parameter is a public instance fields._
+_The parameter is a public instance field._
 
 #### `selector`
 
 `String`
 
 The CSS selector used in the template to identify the step.
 
-_The parameter is a public instance fields._
+_The parameter is a public instance field._
 
 #### `optionalStep`
 
 `Boolean`
 
-Declare if the step is optional and can be submit without validation. The validation is set on the `canTheStepBeDisplayed` function.
+To declare if the step is optional and can be submit without validation. The validation is set on the `canTheStepBeDisplayed` function.
```

Comment:
_...can be submitted..._

Dsl j'avais pas vu sur la first review =/

Reply:
C'est corrigé